cime_config/allactive/config_compsets.xml

```diff
@@ -113,12 +113,17 @@
 
 <compset>
   <alias>WCYCLSSP585</alias>
-  <lname>SSP585SOI_EAM%CMIP6_ELM%SPBC_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
+  <lname>SSP585SOI_EAM%CMIP6_ELM%CNPRDCTCBCTOP_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
 </compset>
```

Comment:
Is there any particular consideration to name these SSPs under WCYCL? These compsets are now being created for different groups.

Reply:
@hwangacme , that is more of continuing the naming convention we have been having. We are still a distinction among WCYCLSSP, CRYOSSP, BGCSSP.